util/searchRankScript.bash

```diff
@@ -48,26 +50,43 @@ fi
 [[ -d "$exampleDir" ]] && score=$((score+1)) && echo "example dir found in package"
 [[ -d "$srcDir" ]] && score=$((score+1)) && echo "src dir found in package"
 
-# points for number of examples 
+# points for number of examples (max 3)
 countForExamples=$(ls -l $exampleDir/*.chpl 2>/dev/null | wc -l)
 echo "Found $countForExamples examples in package."
+maxScoreExamples=3
+if [ "$countForExamples" -gt 3 ] 
+then 
+  countForExamples=$maxScoreExamples
+fi
 score=$((score+countForExamples))
 
-# points for number of tests
+# points for number of tests (max 3)
+maxScoreTests=3
 countForTests=$(ls -l $testDir/*.chpl 2>/dev/null | wc -l)
 echo "Found $countForTests tests in package."
+if [ "$countForTests" -gt 3 ]
+then
+  countForTests=$maxScoreTests
+fi
 score=$((score+countForTests))
 echo "Total score generated for $name = $score"
 
+# normalize the score to 100
+maxScore=$((6+maxScoreTests+maxScoreExamples))
```

Comment:
Trying to eliminate magic numbers... Where does the 6 come from, and can we define that programmatically like we did for the `maxScoreTests` and `maxScoreExamples`?